INSTALL.md

```diff
@@ -93,7 +93,7 @@ If you are a Node.js developer, you may wish to install the balena CLI via [npm]
 The npm installation involves building native (platform-specific) binary modules, which require
 some additional development tools to be installed first:
 
-* [Node.js](https://nodejs.org/) version 8, 10 or 12
+* [Node.js](https://nodejs.org/) version 10 or 12
```

Comment:
I see you've labeled this PR `change-type: major`, resulting in CLI v12.0.0. We've been collecting breaking changes for v12 in this GDoc: https://docs.google.com/document/d/1uHE-_LLu5gJUQZICQJtlo76EQCm90z0CMAwpkfw6s8w/edit

@srlowe, I think pressure is building to release CLI v12... Node v8 is no longer supported, Node 14 is around the corner, Page wanted to take advantage of Node 10+ features, we need to upgrade balena-sdk too, including testing. I wanted to include issue #1032 in v12, but each week there are new hurdles (the latest being balena CI being unhappy with the MIT licence in the migration from `zeit/dockerignore` to `balena-io-modules/dockerignore`). Anyway, we need to put some effort into that GDoc and CLI v12, perhaps with a pause in the oclif/typescript conversion. Maybe something we could also discuss with @OdysLam on Thursday's call.

Reply:
@pdcastro is there a v12 release branch that you PR against?
Also heads up that there is a pending arch call item wrt whether we should be using release branches at all or not, so it would be great if you could be there  as well.

Reply:
No release branch yet. My thinking is that, when possible, we should release the "breaking changes" behind flags (command-line options) that enable the new behaviour in the current major release. This is possible, for example, for the new gitignore / dockerignore behaviour (but probably not possible for Node 8 → Node 14 support). In this case, it is not really a breaking change, and it also allows users to try the new feature before upgrading the CLI to the next major version. Then, when all this work is done and already released, a release branch is created for the shortest amount of time possible, and what it does is to change the "default behaviour" (without the enabling flags) to be same as the behaviour with the enabling flags, and the enabling flags become a no-op (redundant). I'll look out for the arch call, thanks for the hint! 👍